src/Arcus.Templates.ServiceBus.Queue/Program.cs

```diff
@@ -13,25 +13,19 @@ namespace Arcus.Templates.ServiceBus.Queue
 {
     public class Program
     {
-#if (ExcludeSerilog == false)
+#if Serilog
         #warning Make sure that the appsettings.json is updated with your Azure Application Insights instrumentation key.
         private const string ApplicationInsightsInstrumentationKeyName = "TELEMETRY_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
We should align the name of this setting so that it also uses `APPINSIGHTS_INSTRUMENTATIONKEY`

Reply:
Yes, I know, is still WIP (draft PR + 🚧).

Reply:
Done in https://github.com/arcus-azure/arcus.templates/pull/587/commits/844d1b8d5f71c36dc7c34c097bf47629a2088bd1